packages/lesswrong/components/form-components/ImageUpload2.tsx

```diff
@@ -0,0 +1,177 @@
+  classes: ClassesType
+}) => {
+  const theme = useTheme();
+
+  const setImageInfo = (error, result) => {
+    if (error) {
+      throw new Error(error.statusText)
+    }
+    // currently we ignore all events other than a successful upload -
+    // see list here: https://cloudinary.com/documentation/upload_widget_reference#events
+    if (result.event !== 'success') {
+      return
+    }
+    const imageInfo = result.info
+    if (imageInfo && imageInfo.public_id) {
+      setImageId(imageInfo.public_id)
+      updateCurrentValues({[name]: imageInfo.public_id})
+    } else {
+      //eslint-disable-next-line no-console
+      console.error("Image Upload failed");
+    }
+  }
+
+  const uploadWidget = () => {
+    const cloudinaryArgs = cloudinaryArgsByImageType[name]
+    if (!cloudinaryArgs) throw new Error("Unsupported image upload type")
+    // @ts-ignore
+    cloudinary.openUploadWidget({
+      multiple: false,
+      sources: ['local', 'url', 'camera', 'facebook', 'instagram', 'google_drive'],
+      cropping: true,
+      cloudName: cloudinaryCloudNameSetting.get(),
+      theme: 'minimal',
+      croppingValidateDimensions: true,
+      croppingShowDimensions: true,
+      styles: {
+        palette: {
+            tabIcon: theme.palette.primary.main,
+            link: theme.palette.primary.main,
+            action: theme.palette.primary.main,
+            textDark: "#212121",
+        },
+        fonts: {
+            default: null,
+            "'Merriweather', serif": {
+                url: "https://fonts.googleapis.com/css?family=Merriweather",
+                active: true
+            }
+        }
+      },
+      ...cloudinaryArgs,
+      ...(croppingAspectRatio ? {croppingAspectRatio} : {})
+    }, setImageInfo);
+  }
+  
+  const removeImg = () => {
+    clearField()
+    setImageId(null)
+  }
+
+  const [imageId, setImageId] = useState(() => {
+    if (document && document[name]) {
+      return document[name];
+    }
+    return ''
+  })
+  
+  const formPreviewSize = formPreviewSizeByImageType[name]
+  if (!formPreviewSize) throw new Error("Unsupported image upload type")
+  
+  const imageUrl = imageId ? makeCloudinaryImageUrl(imageId, {
+    c: "fill",
+    dpr: "auto",
+    q: "auto",
+    f: "auto",
+    g: "auto:faces"
+  }) : placeholderUrl
+  
+  return (
+    <div className={classes.root} {...formPreviewSize}>
+      <Helmet>
+        <script src="https://upload-widget.cloudinary.com/global/all.js" type="text/javascript" />
+      </Helmet>
+      <div className={classes.imageBackground} style={{ backgroundImage: `url(${imageUrl})` }}>
+        <div className={classes.buttonRow}>
+          <Button onClick={uploadWidget} className={classNames("image-upload-button", classes.button)}>
+            {imageId ? `Change` : `Upload ${label}`}
+          </Button>
+          {imageId && <Button className={classes.button} title="Remove" onClick={removeImg}>
+            Remove
+          </Button>}
```

Comment:
I think the button colors here aren't quite right in dark mode (comparing with the design [here](https://www.figma.com/file/DF9TLXQJuPrEJYEesS05J0/Social-preview?node-id=81%3A69&t=ojEM6DTRCraDhnen-0)):

<img width="651" alt="Screen Shot 2023-01-27 at 1 31 13 AM" src="https://user-images.githubusercontent.com/9057804/215023900-440a2ddd-db97-4946-8707-350790ab2714.png">

Reply:
Also the design calls for a few other smaller tweaks, like changing the section border color and adding a border radius - I assume you're intentionally excluding the yellow points for now?